ci-ocio/Dockerfile

```diff
@@ -12,6 +12,7 @@ FROM ${ASWF_PKG_ORG}/ci-package-glew:${VFXPLATFORM_VERSION} as ci-package-glew
 FROM ${ASWF_PKG_ORG}/ci-package-glfw:${VFXPLATFORM_VERSION} as ci-package-glfw
 
 FROM ${ASWF_PKG_ORG}/ci-package-openexr:${VFXPLATFORM_VERSION} as ci-package-openexr
+FROM ${ASWF_PKG_ORG}/ci-package-ocio:${VFXPLATFORM_VERSION} as ci-package-ocio
```

Comment:
@aloysbaillet I think I'm confused on what the purpose of the ci-ocio or ci-openexr images actually is. I thought it would be the combination of all the packages required to run the particular software but looking at the original Dockerfiles they all seem to import ci-common as the named software:
```FROM ${ASWF_ORG}/ci-common:${CI_COMMON_VERSION} as ci-ocio```

This results in an image that doesn't actually contain the named software but contains everything except the software in question which would then need to be downloaded and compiled again, even though we've already built said software in the packages. Is that correct?

Reply:
The primary purpose of these containers it to provide a VFX Reference Platform compliant, containerized build environment for the ASWF projects which can be used by their respective CI pipelines. The layered hierarchy is roughly:

* `ci-common` has base CentOS 7 environment and developer tools
* `ci-base` adds pre-built versions of the non-ASWF components specified by the VFX Reference Platform such as Boost, Qt  or TBB
* `ci-ocio` for instance adds pre-built dependencies that are required specifically to build the OCIO project
* `ci-vfxall` contains pre-built versions of everything specified by the VFX Reference Platform

So indeed, the idea is that `ci-ocio` shouldn't contain a pre-built version of OCIO since its purpose is to provide all the pre-built dependencies that the `ocio` project needs for nightly and release builds.

`aswf-docker` is not currently meant to be a mechanism to release versioned artifacts from the ASWF projects, right now that's being done ad hoc per project. If someone wanted a build container to build against all the versions specified by a specific year of the VFX Reference Platform, the `ci-vfxall:20XX` container could be used for that purpose.

(@aloysbaillet can correct me if I got this wrong)